components/raftstore/src/store/peer.rs

```diff
@@ -892,6 +892,10 @@ where
     pub snapshot_recovery_state: Option<SnapshotBrState>,
 
     last_record_safe_point: u64,
+    /// Used for checking whether the peer is pending on apply.
+    /// If None, the peer has no pending logs for apply or already
+    /// finishes applying.
+    pub pending_on_apply: Option<bool>,
```

Comment:
What do `Some(true)` and `Some(false)` mean?

Reply:
Re:
* `Some(false)` => initial state, peer is build at start. Not be recorded into `pending_peers`.
* `Some(true)` => peer is on applying logs state, and already recorded into `pending_peers`.